waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -31,10 +31,12 @@ contract(MembershipContract):
   # TODO define a return type of bool for register method to signify a successful registration
   proc register(pubkey: Uint256) # external payable
 
-proc membershipKeyGen*(): Option[MembershipKeyPair] =
-  # generates a MembershipKeyPair that can be used for the registration into the rln membership contract
-  var 
-    merkleDepth: csize_t = 32
+proc createRLNInstance*(d: int, ctxPtrPtr: ptr (ptr RLN[Bn256])): bool = 
```

Comment:
ctxPtrPtr param looks a bit gnarly, can this be simplified/encapsulated better somehow with a Nim reference? Or isolate this

Reply:
For sure! will investigate it in this issue https://github.com/status-im/nim-waku/issues/433